parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -122,7 +122,7 @@ configureCustomCaCertificate() {
         declare varname=CUSTOM_CA_CERT_${i} 
         echo "${!varname}" > /opt/certs/00000000000000cert${i}.crt
     done
-    systemctl restart update_certs.service || exit $ERR_UPDATE_CA_CERTS
+    systemctl restart update_certs_provisioning.service || exit $ERR_UPDATE_CA_CERTS
```

Comment:
ah this is annoying. it's not guaranteed to be on disk for older vhds. can you avoid changing the name

Reply:
wait, this is an addition to the previous service? you don't need this at all...